WcaOnRails/app/views/shared/_associated_events_picker.html.erb

```diff
@@ -23,7 +23,7 @@ Note: form_builder.object is the object having associated events.
 
 <div class="form-group <%= "has-error" unless errors.empty? %>">
   <%= label_tag events_association_name, class: "associated-events-label" do %>
-    <%= label %>
+    <%= label %> (<span class="events-selected-count"></span> selected)
```

Comment:
Did you see the latest code? The key is now `count_selected: "%{count} selected"`, so translators should have the flexibility to choose a phrasing that works for their language.